quesma/frontend_connectors/tcp_mysql_connection_handler.go

```diff
@@ -0,0 +1,107 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+
```

Comment:
Maybe we should add a comment.
```suggestion
// Experimental alpha frontend for Mysql protocol

```